sql/core/src/main/scala/org/apache/spark/sql/execution/python/BatchEvalPythonExec.scala

```diff
@@ -62,6 +62,7 @@ import org.apache.spark.util.Utils
  */
 case class BatchEvalPythonExec(udfs: Seq[PythonUDF], output: Seq[Attribute], child: SparkPlan)
```

Comment:
Maybe rename `BatchEvalPythonExec` as it is not just for batched python udfs now.

Reply:
How about `BlockedEvalPythonExec` or something?

Reply:
I feel it is better than `BatchEvalPythonExec`. Don't know if others have any suggestion.

Reply:
Thanks! Let's see if others have any suggestions for a while.

Reply:
`BlockedEvalPythonExec` sounds better to me too.